### PR TITLE
Fix wording about CSM

### LINKED ISSUE #3302: Remove "contact success manager" from runtime support
### Issue Type

Inaccurate, misleading, or out-of-date information

### Links to Affected Docs

https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#backport-policy-for-bug-and-security-fixes

### The Issue or Context

"Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager." 

### Fix or Proposed Change

Most users don't have a specific CSM. Instead, will add a link to https://docs.astronomer.io/astro/astro-support

### Additional Notes

_No response_

### Required Reviewers

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3302

### CODE REVIEW

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ If you identify a vulnerability that results in relevant risk for your organizat
 
 - **Security vulnerabilities:** When Astronomer identifies a significant security vulnerability in Astro Runtime, a fix is backported and made available as a patch version for all stable and LTS versions in maintenance. A significant security issue is defined as an issue with significant impact and exploitability.
 
-Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
+Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, see how to [Submit a support request](https://docs.astronomer.io/astro/astro-support).
```
Comment (collinmcnulty) [inline astro/runtime-version-lifecycle-policy.md:70]:
```suggestion
Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions.
```

File: software/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ If you identify a vulnerability that results in relevant risk for your organizat
 
 - **Security vulnerabilities:** When Astronomer identifies a significant security vulnerability in Astro Runtime, a fix is backported and made available as a patch version for all stable and LTS versions in maintenance. A significant security issue is defined as an issue with significant impact and exploitability.
 
-Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
+Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, see how to [Submit a support request](https://docs.astronomer.io/astro/astro-support).
```
Comment (collinmcnulty) [inline software/runtime-version-lifecycle-policy.md:69]:
```suggestion
Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions.
```

File: software_versioned_docs/version-0.30/runtime-version-lifecycle-policy.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ When Astronomer identifies a significant bug in Astro Runtime, a fix is backport
 
 When Astronomer identifies a significant security vulnerability in Astro Runtime, a fix is backported and made available as a patch version for all stable and LTS versions in maintenance. A significant security issue is defined as an issue with significant impact and exploitability.
 
-Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
+Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, see how to [Submit a support request](https://docs.astronomer.io/astro/astro-support).
```
Comment (collinmcnulty) [inline software_versioned_docs/version-0.30/runtime-version-lifecycle-policy.md:60]:
```suggestion
Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions.
```

File: software_versioned_docs/version-0.32/runtime-version-lifecycle-policy.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ If you identify a vulnerability that results in relevant risk for your organizat
 
 - **Security vulnerabilities:** When Astronomer identifies a significant security vulnerability in Astro Runtime, a fix is backported and made available as a patch version for all stable and LTS versions in maintenance. A significant security issue is defined as an issue with significant impact and exploitability.
 
-Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, contact your customer success manager.
+Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions. To request a fix for a specific bug, see how to [Submit a support request](https://docs.astronomer.io/astro/astro-support).
```
Comment (collinmcnulty) [inline software_versioned_docs/version-0.32/runtime-version-lifecycle-policy.md:69]:
```suggestion
Occasionally, Astronomer might deviate from the defined response policy and backport a bug or security fix to releases other than the latest stable and LTS versions.
```

**Review by collinmcnulty — CHANGES_REQUESTED** [review]
This additional backporting is really at Astronomer's discretion, and I don't want to give the impression that a support ticket requesting a backport is likely to change that.

**Review by jwitz — APPROVED** [review]

**Pull request merged**